website/events/models.py

```diff
@@ -28,7 +28,7 @@ class Event(Model, metaclass=ModelTranslateMeta):
         verbose_name=_("image"), null=True, blank=True, on_delete=SET_NULL
     )
     start_date = DateTimeField(verbose_name=_("start date"))
-    end_date = DateTimeField(verbose_name=_("end date"))
+    end_date = DateTimeField(null=True, blank=True, verbose_name=_("end date"))
```

Comment:
In de view van de events, moet je ook inbouwen wat er gebeurt als hij geen eind datum heeft. En ook in de checks van de registreer view enzo moet je dit aanpassen.